packages/block-editor/src/components/rich-text/use-paste-handler.js

```diff
@@ -222,3 +225,17 @@ export function usePasteHandler( props ) {
 		};
 	}, [] );
 }
+
+/**
+ * Normalizes a given string of HTML to remove the Windows specific "Fragment" comments
+ * and any preceeding and trailing whitespace.
+ *
+ * @param {string} html the html to be normalized
+ * @return {string} the normalized html
+ */
+function removeWindowsFragments( html ) {
+	const startReg = new RegExp( '.*<!--StartFragment-->', 's' );
```

Comment:
Nitpick: I think we prefer to use regex literal when creating regex?

```suggestion
	const startReg = /.*<!--StartFragment-->/s;
```

Reply:
Fixed